src/Calculator.Shared/Styles.xaml

```diff
@@ -167,7 +167,7 @@
 									   Color="#17000000" />
 
 			<xamarin:SolidColorBrush x:Key="AppControlHoverButtonFaceBrush"
-									 Color="{ThemeResource SystemColorHighlightColor}" />
```

Comment:
Why did you explicitly specify the color? Won't that break themeing ?

Reply:
I copied the `win:RevealBrush` colour directly above. It's the same as is used in the original repo https://github.com/microsoft/calculator/blob/1a9edff26b3fb8f7a1dcdf13e860da7794d4cf13/src/Calculator/App.xaml#L83

It's good you mentioned theming though, I forgot to make the same change for the Dark (aka Default) resources, just fixed that

Reply:
Sweet ;)